### ISSUE TITLE
Bug Report: [oneclick_link] not working

### ISSUE THREAD

**Issue by Fssanc** [stcr/subscribe-to-comments-reloaded#264]
As the title claims, I cannot achieve to show the One-click Unsubscribe link. When I put it as its explained at GitHub:

> Add the Tag [oneclick_link] to the Comment Form Subscribed label to allow the users to Unsubscribe on the same post without going to the Management Page.

It just shows the shortcode itself… I already uninstalled the whole plugin and the problem is still there... any ideas?

Thank you for your time and this great work!

**Comment by Reedyseth** [comment]
@Fssanc , Make sure, to add the tag in the notification message, that is the only place where it works.

![image](https://cloud.githubusercontent.com/assets/1228711/18568165/0a884a84-7b59-11e6-908b-eaed8f2fe484.png)

**Comment by Fssanc** [comment]
Oooh... it cannot be used on any other spot?? It would be perfect to use it right after the comments form, where you subscribed... there is not a way to find the function to build the link? I can add the parameters or even create a form to unsubscribe, but i do not know where to find the parameters to send... :S

Could you help me with this? Thank you again, @Reedyseth !

**Comment by Reedyseth** [comment]
> Could you help me with this? Thank you again, @Reedyseth !

Yes, as a matter of fact I have created a ticket https://github.com/stcr/subscribe-to-comments-reloaded/issues/185 with this request. This option will be ready for the next release.

**Comment by Fssanc** [comment]
Alright! I actually thought it was already implemented... x)

So, thanks one more time for your nice work with this plugin, its a really good job!

P.S.: I will try to help a little bit translating some parts that remain in english to spanish! ;)

**Comment by Reedyseth** [comment]
Gracias, I will appreciate !
